website/blog/2022-07-04-aluminium.md

```diff
@@ -0,0 +1,478 @@
+2. Metals will check before starting if your Java home is correct, and if it's
+   not it will replace it with the current one used by Metals.
+3. Metals will not show any notifications if it cannot change the file itself.
+
+If you encounter any further issues, you can take a look at Metals output and it
+should show logs like these:
+
+```
+2022.07.01 16:14:41 INFO  Bloop uses /usr/lib/jvm/openjdk17 defined at /home/tgodzik/.bloop/bloop.json
+2022.07.01 16:14:41 INFO  Bloop currently uses settings: -Xmx3G
+```
+
+You can use them to debug what is happening and report an issue.
+
+## Allow to run all scalafix rules on a file
+
+Up until the current release Metals would only run scalafix for
+`organize imports` and wouldn't allow to use any other rules. Starting with this
+release you should be able to apply all currently defined
+`rules in .scalafix.conf` you require by running `metals.scalafix-run` command.
+
+By default it's bound to `alt` + `ctrl` + `shift` + `o` shortcut in Visual
+Studio Code.
+
+![scalafix-run](https://i.imgur.com/DLe5sYT.gif)
+
+One major drawback of the current approach is that we are unable to run some of
+the custom rules. Only the default rules and the contributed Hygiene rules are
+currently available by default. You can see the full list of them
+[here](https://scalacenter.github.io/scalafix/docs/rules/community-rules.html#hygiene-rules).
+For all the rest users will have to use a previously added setting
+`metals.scalafixRulesDependencies`, where you can add dependencies following the
+[coursier](https://get-coursier.io/) convention such as
+`com.github.liancheng::organize-imports:0.6.0`.
+
+Let us know if there is another rule we should add to the default ones!
+
+The current situation could be improved by making sure that the dependencies for
+scalafix are listed in the configuration itself. This would allow both Metals
+and scalafix CLI to easily run rules on your workspace. You can find the
+discussion on
+[scalafix github page](https://github.com/scalacenter/scalafix/issues/1625)
+
+## [Scala 3] Implement-all abstract members code action.
+
+Implement-all abstract members code action is finally available in Scala 3 🎉
+
+Previously, in Scala3, when you were implementing an abstract member in a
+concrete class, you could only see the completion to override particular
+members. However, you needed to add those members one by one. Now you'll see a
+code action to implement all the members at once.
+
+![implement-all](https://i.imgur.com/foU3oHL.gif)
+
+Great work by [tanishiking](https://github.com/tanishiking).
+
+## [Scala 3] Add file name completions
+
+A file name completion feature has been available only in the Scala 2 project,
+but thanks to the effort by [riiswa](https://github.com/riiswa), Metals 0.11.7
+will complete class name based on the enclosing file, in Scala3!
+
+![filename-completion](https://i.imgur.com/f10pnJ6.gif)
+
+## [Scala 3] Expression evaluation for debugger.
```

Comment:
Added this section additionally, since we are ready for people to start testing the expression evaluator.